### ISSUE TITLE
on ios simulator diffrent tab with one scroll offset

### ISSUE THREAD

**Issue by looveu** [brentvatne/react-native-scrollable-tab-view#99]
my code is

```
<View style={styles.container}>
        <ScrollableTabView>
          <ScrollView tabLabel="zero" style={styles.tabView}>
                  something1
          </ScrollView>
          <ScrollView tabLabel="one" style={styles.tabView}>
                 something2
          </ScrollView>
          <ScrollView tabLabel="two" style={styles.tabView}>
                  something3
          </ScrollView>
          <ScrollView tabLabel="three" style={styles.tabView}>
            something4
          </ScrollView>
          <ScrollView tabLabel="four"  style={styles.tabView}>
            something5
          </ScrollView>
        </ScrollableTabView>
      </View>
```

on ios simulator when i scroll someone tab the other tab is scrolled too.

**Comment by antihate** [comment]
Having the same issue.. tested on latest release with RN 0.16.0 & 0.17.0

![ios-scroll-bug](https://cloud.githubusercontent.com/assets/2887867/11918701/1d3d1116-a75e-11e5-8e53-60c81ee80156.gif)

**Comment by kfiroo** [comment]
I have the same problem, this commit seems to resolve it.
https://github.com/brentvatne/react-native-scrollable-tab-view/commit/58c9f1474993a054ced86d3e99fc8fa4505313fa

**Comment by skv-headless** [comment]
fixed #113
